Fix GHA - Release Turtles against rancher/rancher

Currently failing due to error, `make: *** No rule to make target '.dapper'.`
https://github.com/rancher/turtles/actions/runs/25306202341/job/74188586415#step:11:16

Inputs given to [CI workflow](https://github.com/rancher/turtles/actions/workflows/release-against-rancher.yml) in above run:
Use workflow from - `main`
Submit PR against the following rancher/rancher branch (e.g. release/v2.14). Use 'auto' to derive from the current turtles branch - `main`
Previous Turtles version - `v0.26.0`
New Turtles version - `v0.26.1`
Set 'true' to bump chart major version when the Turtles minor version increases (e.g., v0.23.0 → v0.24.0-rc.0) - `false`

--- a/.github/scripts/release-against-rancher.sh
+++ b/.github/scripts/release-against-rancher.sh
@@ -132,12 +132,7 @@ fi
 
 yq --inplace ".turtlesVersion = \"${NEW_CHART_VERSION}+up${NEW_TURTLES_VERSION_SHORT}\"" ./build.yaml
 
-# Downloads dapper
-make .dapper
-
-# DAPPER_MODE=bind will make sure we output everything that changed
-DAPPER_MODE=bind ./.dapper go generate ./... || true
-DAPPER_MODE=bind ./.dapper rm -rf go .config
+go generate ./...
 
 git add .
 git commit -m "$COMMIT_MSG"

